Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/ComponentModeSwitcher.h

```diff
@@ -34,11 +34,13 @@ namespace AzToolsFramework
 
             // Member variables
             AZ::EntityComponentIdPair m_pairId; //!< Id of entity component pair.
-            AZ::Entity* m_entity = nullptr; //!< Pointer to entity associated with pairId.
-            AZ::Component* m_component = nullptr; //!< Pointer to component associated with pairId.
+            //AZ::Entity* m_entity = nullptr; //!< Pointer to entity associated with pairId.
+            //AZ::Component* m_component = nullptr; //!< Pointer to component associated with pairId.
```

Comment:
Please remove these instead of commenting them out

Reply:
Thank you for this :)